### ISSUE TITLE
implement CRT shader effects

### ISSUE THREAD

**Issue by nippur72** [floooh/chips-test#13]
I've found a comment in the source code mentioning CRT effects so I would like to point you to this very cool shader effect... it has scanlines, rgbmask and pincushion:

[https://www.shadertoy.com/view/XsjSzR](https://www.shadertoy.com/view/XsjSzR)

It doesn't need to be included in the C source code, it can be simply applied to the HTML canvas.

**Comment by floooh** [comment]
Yep I know that effect, and I even had it implemented in a previous emulator.

See here and click on the little "CRT" button in the bottom left corner:

http://floooh.github.io/virtualkc/

In this older emulator I had to make the effect optional because performance on mobile devices was very poor, so I tried to come up with cheaper effects in the new emulators, but this ended up in lots of tinkering without satisfying results, so I switched it off again to maybe do it "properly" later.

**Comment by nippur72** [comment]
WOW that is absolutely amazing, it looks like a true CRT display! I wonder if there's a simple way to apply it to a HTML canvas, because I don't know much about shaders & c.

**Comment by chrisbward** [comment]
+1 would love to see some shaders in the mix!
